src/main/resources/feedbackQuestionConstSumEditFormOptionFragment.html

```diff
@@ -1,5 +1,5 @@
-<div id="constSumOptionRow-${i}-${questionNumber}">
-    <div class="input-group col-sm-12">
+<div class="margin-bottom-7px" id="constSumOptionRow-${i}-${questionNumber}">
```

Comment:
then we have to change them all. I think it is highly unlikely that we would need another 15px bottom margin in an unrelated page. But it is very possible that we might have to change that value to 16px or 20px. Am I right to say that?